### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by benib**

### CODE REVIEW

**Review by RezaS — COMMENTED** [review]
Best review ever

File: test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+I am a test.
```
Comment (albanbochsler) [inline test.txt]: verbuggt

**Review by albanbochsler — CHANGES_REQUESTED** [review]
need major bugfixes

File: test.txt (hunk 1 above)
Comment (benib) [reply test.txt]: nei isch scho guet so

**Review by albanbochsler — APPROVED** [review]
nice work. go

**Review by albanbochsler — APPROVED** [review]

**Pull request merged**